taint/test/mochitest/mochitest.ini

```diff
@@ -42,6 +42,5 @@ support-files =
 [test_websocket.html]
 [test_push.html]
 [test_dom.html]
+[test_string_convertion_with_classes.html]
 scheme = https
-[test_url_object.html]
-[test_message_event.html]
```

Comment:
Were these tests removed on purpose? If not could you add them back in please?

Reply:
Sorry about that, I did not realize these tests were removed. I added them back.